src/java.security.jgss/share/classes/sun/security/krb5/internal/KDCRep.java

```diff
@@ -23,16 +23,17 @@
  */
 
 /*
- *
- *  (C) Copyright IBM Corp. 1999 All Rights Reserved.
- *  Copyright 1997 The Open Group Research Institute.  All rights reserved.
+ * ===========================================================================
+ * (c) Copyright IBM Corp. 1999, 2023 All Rights Reserved
```

Comment:
This IBM copyright is from an IBM contribution to OpenJDK, not changes made for the OpenJ9 project. I think we can just leave it as-is, adding an end date is not required. @keithc-ca thoughts?

Reply:
I'm fine with leaving this alone. We've stopped adding a second year to copyrights in related projects so we could do the same here.

Reply:
Reverted to original.